File: TRClient/src/app/Entities/CarbonFuelTax.ts
export class CarbonFuelTax  {
    Completed:boolean;
    Jurisdiction: string
    AmtFreightTrain: number;  
    AmtWorkTrain: number;  
    AmtYardSwitching: number;  
    AmtTotalLine: number;  
    Adjustment: number;  
    TotalLitresToPay: number;  
    RatePerLitre: number;  
    TotalToPay: number;  
    PercDiff: number;  
    TotalLitres: number;
    IsOdd: boolean;
    constructor() { 
        this.Completed = false;
        this.Jurisdiction = '';
        this.AmtFreightTrain = 0;
        this.AmtWorkTrain = 0;
        this.AmtYardSwitching = 0;
        this.AmtTotalLine = 0;
        this.Adjustment = 0;
        this.TotalLitresToPay = 0;
        this.RatePerLitre = 0;
        this.TotalToPay = 0;
        this.PercDiff = 0;
        this.TotalLitres = 0;
        this.IsOdd = false;
     }
 }

File: TRBusinessLayer/DataObjects/CarbonFuelTax.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace TRBusinessLayer.DataObjects
{
    public class CarbonFuelTax
    {
        public bool Completed { get; set; }
        public string Jurisdiction { get; set; }
        public decimal AmtFreightTrain { get; set; }
        public decimal AmtWorkTrain { get; set; }
        public decimal AmtYardSwitching { get; set; }
        public decimal AmtTotalLine { get; set; }
        public decimal Adjustment { get; set; }
        public decimal TotalLitresToPay { get; set; }
        public decimal RatePerLitre { get; set; }
        public decimal TotalToPay { get; set; }
        public decimal PercDiff { get; set; }

        public decimal TotalLitres{get; set;}
    }
}